2nd-gen/packages/swc/components/badge/test/badge.a11y.spec.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2025 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import AxeBuilder from '@axe-core/playwright';
+import { expect, test } from '@playwright/test';
+
+import { gotoStory } from '../../../../../test/a11y-helpers.js';
```

Comment:
Given that this is `@adobe/swc` related-only, it feels like this helper could be moved `2nd-gen/packages/swc/utils`